ambari-server/src/main/java/org/apache/ambari/server/orm/entities/UserEntity.java

```diff
@@ -211,11 +208,11 @@ public void setLocalUsername(String localUsername) {
     this.localUsername = localUsername;
   }
 
-  public Date getCreateTime() {
+  public long getCreateTime() {
     return createTime;
   }
 
-  public void setCreateTime(Date createTime) {
+  public void setCreateTime(long createTime) {
```

Comment:
Can't we keep the original signature by doing the conversion inside?

Reply:
I believe it's better to return what the actual column type represents and let the clients decide how to convert.